Gothic-Unity-Project/Gothic-Unity
# Highlight objects pointed at + show its name

Like G1, marked objects are now brightened up and their name is shown.

To test:
* General expected behaviour: Objects should be highlighted most of the time when pointed towards.
* Spawn Lab with HVR (+/- Simulator) and test the oCMob* elements left of ladder demo.
* Spawn world and try if names and highlights are shown on objects as well
    * Object names should rotate along your camera movement
* GameConfiguration has two new fields: _BrightenUpHoveredVOBs_ + _ShowNamesOnHoveredVOBs_ -> Please disable them and see if it works as you expect it.

Boundary:
* Inside lab, VOBs have no name
* Some objects also have no name inside G1 (e.g. lights on the street). It's intended behavior by G1.
* HVR "Hover" feature sometimes doesn't recognize, that we point to an object. We need to investigate it later if it disturbs.
* It might be, that not all VOB names are perfectly visible from all angles. Please judge for yourself if it's a blocker or ok for now.
* Separate tickets:
    * Doors and Chests have the HVRPhysicalDoor component and rotate already. But it's still physics based and will stop when a world collision (e.g. a wall) is detected. I will mention this in the Door/Chest tickets to solve there. (#19 + #23)
    * Every object has HVRGrabbable as component to use its Hover events. Even if we won't "grab" an object (e.g. a lever). I will create a separate ticket to think about a better logic. (#67)
    * Some objects (e.g. water pipe) have incorrect Grabbable logic (prefab-ed on wrong spot). We will create a new ticket for better lookup during mesh creation in a separate ticket. (#68)
